crates/rust-analyzer/src/to_proto.rs

```diff
@@ -164,27 +164,27 @@ pub(crate) fn completion_item(
         additional_text_edits: Some(additional_text_edits),
         documentation: completion_item.documentation().map(documentation),
         deprecated: Some(completion_item.deprecated()),
-        command: if completion_item.trigger_call_info() {
-            let cmd = lsp_types::Command {
-                title: "triggerParameterHints".into(),
-                command: "editor.action.triggerParameterHints".into(),
-                arguments: None,
-            };
-            Some(cmd)
-        } else {
-            None
-        },
         ..Default::default()
     };
 
     if completion_item.score().is_some() {
-        res.preselect = Some(true)
+        res.preselect = Some(true);
```

Comment:
I think only one item can be preselected in the reply. I'm not sure we enforce that yet.

Reply:
Many items can, but that doesn't matter, as only the first one will be selected.